composer.json

```diff
@@ -29,8 +29,9 @@
         "phpunit/php-file-iterator": "^2.0",
         "phpunit/phpunit": "^7.0",
         "squizlabs/php_codesniffer": "^3.4",
+        "symfony/contracts": "^1.1",
         "symfony/dependency-injection": "^4.3",
-        "symfony/event-dispatcher": "^2.5"
+        "symfony/event-dispatcher": "^2.5|^5.0"
```

Comment:
This is for retro compatibilty for user that decides to update event dispatcher to 5.0